src/include/migraphx/program.hpp

```diff
@@ -139,6 +141,8 @@ struct program
     friend bool operator==(const program& x, const program& y);
     friend bool operator!=(const program& x, const program& y) { return !(x == y); }
 
+    module& get_main_module() { return *this; }
```

Comment:
Can this be added to our python/C API?

Reply:
Thanks. Looks like I forgot. just added. will add corresponding unit tests tomorrow.

Reply:
This should return `module*` not `module&`.

Reply:
fixed